packages/graphql/src/resolvers/IdentityEvents.js

```diff
@@ -105,6 +112,18 @@ export function identity({ id, ipfsHash }) {
       .filter(n => n)
       .join(' ')
 
+    // Make old style embedded avatars access by their IPFS hash.
+    if (identity.avatarUrl == undefined && identity.avatar != 'undefined') {
```

Comment:
nit: use === and !=== operator to be consistent with the rest of our code base.

Same comment for the backfill script code.